src/main/java/io/quarkus/gizmo2/impl/constant/ConstantImpl.java

```diff
@@ -35,8 +35,19 @@ public static StringConstant of(final String value) {
     }
 
     public static ConstantImpl of(Constable constable) {
+        Objects.requireNonNull(constable, "constable");
         if (constable instanceof ConstantImpl con) {
             return con;
+        } else if (constable instanceof Boolean val) {
+            return of(val);
+/*
+        } else if (constable instanceof Byte val) {
+            return of(val);
+        } else if (constable instanceof Short val) {
+            return of(val);
+        } else if (constable instanceof Character val) {
+            return of(val);
+*/
```

Comment:
What happened here? Was this a test?

Reply:
Ouch! Indeed this was a test. Sorry about that, fixed.